### PR TITLE
Update Google Cloud Storage bucket classes.

### PR DESCRIPTION
**Pull request by y33zhang**
product refresh

### CODE REVIEW

**Comment by y33zhang** [comment]
I work at google (yyzhang@google.com).  please grant me cla.  thanks

**Comment by jonparrott** [comment]
@y33zhang for CLA bot to recognize you, you need to register your github username at go/github and use your @google.com address to author your commits.

**Comment by y33zhang** [comment]
ahh, noob github user here ;).  Is there a way to retroactively set my google.com address on previous commits?

**Comment by daspecster** [comment]
@y33zhang, yeah you can `git commit --amend --author="Author Name <email@address.com>"`.

**Comment by y33zhang** [comment]
@daspecster thanks a lot, just amended the commit email address.  Is there anything else I need to do to get past the cla check?

File: storage/unit_tests/test_bucket.py [hunk 1]
```diff
@@ -842,6 +842,27 @@ def test_storage_class_setter_NEARLINE(self):
         self.assertEqual(bucket.storage_class, 'NEARLINE')
         self.assertTrue('storageClass' in bucket._changes)
 
+    def test_storage_class_setter_COLDINE(self):
```
Comment (dhermes) [inline storage/unit_tests/test_bucket.py]: s/COLDINE/COLDLINE

**Comment by dhermes** [comment]
Well that was easy! LGTM modulo the misspelling.

We can merge over the CLA-bot issue.

**Comment by daspecster** [comment]
I couldn't find those new types in the API docs?

**Comment by y33zhang** [comment]
@daspecster - those are getting updated later today

**Comment by y33zhang** [comment]
@dhermes - still can't get cla check to pass.  can you help merge over the cla issue?  thanks!

**Comment by y33zhang** [comment]
@dhermes - can you help provide an ETA on when the docs will be refreshed/updated on the live site?  Thanks!

**Comment by dhermes** [comment]
https://googlecloudplatform.github.io/google-cloud-python/latest/storage-buckets.html#google.cloud.storage.bucket.Bucket.storage_class
https://googlecloudplatform.github.io/google-cloud-python/latest/storage-blobs.html#google.cloud.storage.blob.Blob.storage_class

----

You just made me realize that readthedocs.org is not picking up the correct version (probably due to dependency caching). When I fix that the RTD docs will also be updated at:

http://google-cloud-python.readthedocs.io/en/latest/storage-buckets.html
http://google-cloud-python.readthedocs.io/en/latest/storage-blobs.html

----

Just because the docs are updated doesn't mean the `google-cloud-storage` package on PyPI is. We'd need to do a release for that.

**Pull request merged**